scripts/build.sh

```diff
@@ -10,15 +10,15 @@ if ! which goreleaser >/dev/null; then
     go install github.com/goreleaser/goreleaser@v0.173.2
 fi
 
-if [ -z $ENV ]; then
-    echo "Error: ENV variable must be defined"
+if [ -z $MODE ]; then
+    echo "Error: MODE variable must be defined"
     exit 1
 fi
 
 # Check configuration
 goreleaser check
 
-if [ "$ENV" == "dev" ]; then
+if [ "$MODE" == "dev" ]; then
```

Comment:
I suggest we name this var `SINGLE_TARGET` with boolean value so we can run the script like so in Makefile : 

- Local build: `SINGLE_TARGET=true ./scripts/build.sh` (here ENV is empty and should be default to `dev`)
- Docker build: `SINGLE_TARGET=true ENV=release ./scripts/build.sh`
- Release build: `SINGLE_TARGET=false ENV=release ./scripts/build.sh`

I think running `MODE=dev make release` in the Dockerfile is a bit confusing, building for a single target is not specific to dev environments.

WDYT ?

Reply:
> I think running MODE=dev make release in the Dockerfile is a bit confusing, building for a single target is not specific to dev environments.

This is very relevant, I totally agree on this.
That mean that for the docker build we should do : `ENV=release SINGLE_TARGET=true make build` ?

Reply:
>  That mean that for the docker build we should do : ENV=release SINGLE_TARGET=true make build ?

Yes, but `SINGLE_TARGET` can be set directly in the Makefile

Reply:
Something like that ?

```dockerfile
# Dockerfile
RUN MODE=release ENV=release make build
```


```makefile
# Makefile
build:
    SINGLE_TARGET=true ./scripts/build.sh
release:
    MODE=release ENV=release ./scripts/build.sh
```


```bash
# build.sh
# If ENV is not set, then it should be set to `dev`
# If SINGLE_TARGET is not set, then it should be set to `false`
ENV="${ENV:-dev}"
SINGLE_TARGET="${SINGLE_TARGET:-false}"
```

Reply:
What'd be the purpose of `MODE` here ? I think you can drop it

But yeah that's what I had in mind

Reply:
Yeah sorry we can drop the MODE of course.